.evergreen/run-tests.sh

```diff
@@ -22,9 +22,4 @@ OLD_PATH=$PATH
 PATH=/opt/php/${PHP_VERSION}-64bit/bin:$OLD_PATH
 
 # Run the tests, and store the results in a Evergreen compatible JSON results file
```

Comment:
I assume "store the results in a Evergreen compatible JSON results file" doesn't happen here, nor in PHPC, which uses `run-tests.php`. Should we revise this comment?

Reply:
I was going to look at the result file format and see if this makes test results easier to analyse. Going through the build log to see what's going on isn't really sustainable.